source/localizable/analytics/libraries.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+parent: analytics
+title: libraries
+---
+
+# Libraries <%= edit_link %>
+
+Interested in integrating your eshop with <%= flavor.capitalize %> Analytics?
+
+You may want to use one of our existing libraries, that are developed and supported by us. No need to write any code!
+
+<%= partial 'partials/toc' %>
+
+## Wordpress / WooCommerce
+
+You can find the plugin <%= link_to 'here', 'https://wordpress.org/plugins/skroutz-analytics-woocommerce/', target: '_blank' %>.
+
+Requires to have Wordpress 4.0 or higher and the <%= link_to 'WooCommerce plugin', 'https://wordpress.org/plugins/woocommerce/', target: '_blank' %> installed.
+
+### Automatic installation (recommended)
+
+1. Navigate to your wordpress: `Admin panel > Plugins > Add New`
+2. Search this plugin by its name: *Skroutz Analytics WooCommerce*
+3. Install the plugin.
+4. Activate the plugin.
+
+### Manual installation
+
+1. Download the <%= link_to 'plugin file', 'https://wordpress.org/plugins/skroutz-analytics-woocommerce/', target: '_blank' %> to your computer and unzip it.
+2. Using an FTP client, or your hosting control panel, upload the unzipped plugin folder to your WordPress installation’s directory: `wp-content/plugins/`
+3. Activate the plugin through the Plugins menu in WordPress admin.
+
+### Configuration
+
+1. Navigate to the plugin settings: `Admin panel > WooCommerce > Settings > Integration > Skroutz Analytics`
+2. Specify the `Site` your eshop reports to.
+3. Set the `Shop Account ID` to the plugin's settings.
+4. Specify the `Product ID` that should be sent to analytics.
+
+> ##### Note
+> The product ID must be the same ID used in the XML feed provided to <%= flavor.capitalize %>.
+
+## Related Resources
+
+* Configuration
+  * <%= link_to 'Analytics Settings', 'settings' %>
+  * <%= link_to 'Ecommerce metrics', 'ecommerce' %>
+* <%= link_to 'Examples', 'examples' %>
```

Comment:
The indentation is on purpose like this. There are two sections: Configuration (with two subsections) and Examples. This is how you have it the other pages as well.